### DISCUSSION TITLE
Minecraft - Earth 1:1 Scale

### DISCUSSION THREAD

**Discussion by jason-muljadi** [albertkun/21S-ASIAAM-191A#48] [General]
A cool interesting video about converting the Earth to a 1:1 scale on Minecraft. Kinda talks about map! Would love to hear y'all's thoughts about it.

https://www.youtube.com/watch?v=TG3ycG7Iqjs

**Comment by albertkun** [comment]
I actually tried to set up something similar at the start of COVID-19 to do a 1:1 scale of UCLA when I saw some students working on something similar. Minecraft is an interesting tool that has been used in academia and urban planning a bit.

Even GIS companies have been attempting to make that connection easier:
https://www.geoboxers.com/connecting-3d-models-and-gis-with-minecraft/

Here's an interesting post if you want to read about someone trying to do it too:
https://gis.stackexchange.com/questions/94386/bringing-gis-data-into-minecraft
